tests/integration/options/options_per_project/features/RAPIDOC/01_enabled/test.itest

```diff
@@ -0,0 +1,12 @@
+RUN: %strictdoc export %S --output-dir Output | filecheck %s --dump-input=fail
+CHECK: Published: Rapidoc Openapi test
+
+RUN: %check_exists --file "%S/Output/html/_static/rapidoc/rapidoc-min.js"
+
+RUN: cat %S/Output/html/01_enabled/input.html | filecheck %s --dump-input=fail --check-prefix=CHECK-HTML
+RUN: cat %S/Output/html/01_enabled/input.standalone.html | filecheck %s --dump-input=fail --check-prefix=CHECK-HTML
+RUN: cat %S/Output/html/01_enabled/input-TABLE.html | filecheck %s --dump-input=fail --check-prefix=CHECK-HTML
+RUN: cat %S/Output/html/01_enabled/input-TRACE.html | filecheck %s --dump-input=fail --check-prefix=CHECK-HTML
+RUN: cat %S/Output/html/01_enabled/input-DEEP-TRACE.html 
```

Comment:
`filecheck` is missing

Reply:
I removed filecheck from here because Deep traceability view doesn't contain Openapi table (it has too little size for the table), so the check for containg openapi in this type of document will fail. 
My check only validate the existence of the deep traceability file. Is it okay?

Reply:
Right, I have checked your branch, and it is correct as you have it.